### ISSUE TITLE
[Enhancement] release Releases on Repository of Chocolatey packages 

### ISSUE THREAD

**Issue by 4-FLOSS-Free-Libre-Open-Source-Software** [navossoc/KeePass-Yet-Another-Favicon-Downloader#30]
Hello, this is not an issue in itself. Just an suggestion. Would be nice to have a auto update feature for this primary security applications plugin from trusted source. The sha256 hash of PLRGX can be verified by installation automatically optionally if specified.  _For the lazy ones :) and_ to fix incompatibility issue with future of keepass version updates with plugin by getting all in one place for example.

I don't know if you have the spare time to maintain in parallel the "packages". Which basically would contains a script to URL of github [latest](https://github.com/navossoc/KeePass-Yet-Another-Favicon-Downloader/releases/latest) release asset. 

_If you may want to take a look here if you don't know already:
[keepass plugins packages](https://chocolatey.org/packages?q=keepass&moderatorQueue=&prerelease=true&sortOrder=package-download-count)
(There are many plugins and packages count is growing. )_

**Comment by navossoc** [comment]
Hi, not sure if I understood correctly, but your suggestion is that I provide some mean for 'packagers'  verify the authenticity of the plugin?
Like attach on github an extra file with the sha256 of the file or maybe a gpg asc signature?
Something like that?

Or, if I understood correctly, that I implement an auto update engine for my plugin itself.
If that's the case, I'm not sure about this, after all keepass itself checks for updates, it just does not download.

Do you have any example of another plugin already doing something like this? It may help to clarify your suggestion.

[]'s

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
> Like attach on github an extra file with the sha256 of the file or maybe a gpg asc signature?
Something like that?

wasn't asking if you can sign it with gpg without to much extra effort? 
additionally sha256 in Assets would make it possible for one to check file integrity, yes. BUT [chocolatey](https://chocolatey.org) repository can do this automatically for you (users). :)

> "implement an auto update engine for plugin itself" 

would be overkill. no of keepass itself or plugin does this yet as far i know, only update availability checking.

> Do you have any example of another plugin already doing something like this? It may help to clarify your suggestion.


yes, here are a lot other plugins using this:
[keepass-plugin-favicon](https://chocolatey.org/packages/keepass-plugin-favicon)
> _If you may want to take a look here if you don't know already:
> [keepass plugins packages](https://chocolatey.org/packages?q=keepass&moderatorQueue=&prerelease=true&sortOrder=package-download-count)
> (There are many plugins and packages count is growing. )_
>

**Comment by flcdrg** [comment]
Hi @4-FLOSS-Free-Libre-Open-Source-Software, there is an existing Chocolatey package for this plugin at https://chocolatey.org/packages/keepass-yet-another-favicon-downloader that I maintain.

Is there anything else you wanted that package to do that it doesn't already?
